lib/macho/macho_file.rb

```diff
@@ -264,6 +264,45 @@ def change_rpath(old_path, new_path)
       set_path_in_rpath(rpath_cmd, old_path, new_path)
     end
 
+    # Add the given runtime path to the Mach-O.
+    # @example
+    #  file.rpaths # => ["/lib"]
+    #  file.add_rpath("/usr/lib")
+    #  file.rpaths # => ["/lib", "/usr/lib"]
+    # @param path [String] the new runtime path
+    # @return [void]
+    # @raise [MachO::RpathExistsError] if the runtime path already exists
+    def add_rpath(path)
+      raise UnimplementedError.new("addition of rpaths")
+      raise RpathExistsError.new(path) if rpaths.include?(path)
+    end
+
+    # Delete the given runtime path from the Mach-O.
+    # @example
+    #  file.rpaths # => ["/lib"]
+    #  file.delete_rpath("/lib")
+    #  file.rpaths # => []
+    # @param path [String] the runtime path to delete
+    # @return void
+    # @raise [MachO::RpathUnknownError] if no such runtime path exists
+    def delete_rpath(path)
+      rpath_cmd = command(:LC_RPATH).find { |r| r.path.to_s == path }
```

Comment:
I fully agree on all points. Would you mind adapting the deletion logic accordingly, though this will make the code a bit more complicated? I think this also means we need to make some changes to the `change_rpath` code to perform a few more checks, but not in this PR. Can you put the latter on a TO DO list to address as soon as we have finished the LC creation stuff and adapted existing code to it? (That should be our first priority at the moment.)